resources/core/charts/monitoring/charts/alert-rules/README.md

```diff
@@ -22,9 +23,7 @@ Prometheus uses the a label selector **spec.ruleSelector** to identify those Con
 {{- end }}
 ```
 
-So, to define a new alert rule in Kyma, you need to specify a new ConfigMap.
-
-Best practice is to label the ConfigMaps containing rule files with ```role: alert-rules``` as well as the name of the Prometheus object, ```prometheus: {{ .Release.Name }}```.
+So, to define a **new alert rule in Kyma**, it is needed to create a ConfigMap. And as best practice, ConfigMaps are labelled with ```role: alert-rules``` as well as the name of the Prometheus object, ```prometheus: {{ .Release.Name }}```.

```

Comment:
~So,~ ``To define a new alert rule in Kyma, create a ConfigMap labelled with `role: alert-rules` as well as the name of the Prometheus object as `prometheus: {{ .Release.Name }}`.``